### PR TITLE
Support upload of multiple photos

### PR DESCRIPTION
**Pull request by prashtx**
Process all of the photo-upload questions, not just the first. Ensure that the spinbox plugin gets loaded after its jQuery and jQuery Mobile dependencies.

/cc @hampelm

### CODE REVIEW

File: src/js/form.js [hunk 1]
```diff
@@ -178,22 +178,25 @@ define(function (require) {
...
-        }];
+    var files = fileItems.map(function (i, item) {
+      if (item.files.length > 0) {
+        // Pull out the field name (from the survey form) and the File object.
+        return {
+          fieldName: $(item).attr('name'),
+          file: item.files[0]
+        };
       }
+    }).get();
```
Comment (hampelm) [inline src/js/form.js]: Can you make a note of what the `get` here is doing? My read is that it's jquery's `get` to access the DOM node.

File: src/js/form.js [hunk 2]
```diff
@@ -178,22 +178,25 @@ define(function (require) {
...
+
+    // Remove undefined entries from the array.
+    files = _.compact(files);
```
Comment (hampelm) [inline src/js/form.js]: This is necessary because there might be empty file inputs?

**Comment by hampelm** [comment]
Looks good to merge. Thanks for doing this!

File: src/js/form.js (hunk 2 above)
Comment (prashtx) [inline src/js/form.js]: Yup

File: src/js/form.js (hunk 1 above)
Comment (prashtx) [inline src/js/form.js]: It returns an Array, since calling `.map` on a jQuery object returns another jQuery object.

**Pull request merged**